app/src/main/java/com/example/demo/presentation/CalculationCreateResource.java

```diff
@@ -0,0 +1,27 @@
+package com.example.demo.presentation;
+
+
+public class CalculationCreateResource extends ResourceMethodHandler {
+    public final static String KEY = "POST /calculations";
+
+    public String handle(String content)  {
+        String[] splitedRequestBody = content.split(" ");
+        int number1 = Integer.parseInt(splitedRequestBody[0]);
+        int number2 = Integer.parseInt(splitedRequestBody[2]);
+        String operation = splitedRequestBody[1];
+        String responseContent = "";
+
+        if (operation.equals("+")) {
+            responseContent = String.valueOf(number1 + number2);
+        } else if (operation.equals("-")) {
+            responseContent = String.valueOf(number1 - number2);
+        } else if (operation.equals("*")) {
+            responseContent = String.valueOf(number1 * number2);
+        } else if (operation.equals("/")) {
+            responseContent = String.valueOf(number1 / number2);
+        } else {
+            responseContent = "Error";
+        }
+        return responseContent;
```

Comment:
임시 변수를 쓰지 말고 바로 리턴하세요. 대부분의 경우 `else`는 불필요합니다.

```suggestion
        if (operation.equals("+")) {
            return String.valueOf(number1 + number2);
        }
        if (operation.equals("-")) {
            return String.valueOf(number1 - number2);
        }
        if (operation.equals("*")) {
            return String.valueOf(number1 * number2);
        }
        if (operation.equals("/")) {
            return String.valueOf(number1 / number2);
        }

        // 많은 경우, 여기서 예외를 던집니다.
        return "Error";
```

Reply:
피드백 감사합니다